spec/models/education_benefits_claim_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe EducationBenefitsClaim, type: :model do
+  let(:attributes) do
+    {
+      form: { foo: true }
+    }
+  end
+  subject { described_class.new(attributes) }
+
+  describe "validations" do
+    it { should validate_presence_of(:form) }
```

Comment:
it does seem to add a lot of extra complexity for not much code saved so i dont mind getting rid of it